### DISCUSSION TITLE
Labels

### DISCUSSION THREAD

**Discussion by Courtny** [microsoft/atlas-design#160] [Component]
Labels are used to describe inputs, select menus, textareas, radio buttons, and checkboxes.

**Comment by Courtny** [comment]
Working on a first pass of labels here:

[Labels - Figma](https://www.figma.com/file/K71rlDMFzmB6WiCiGqZV4K/%F0%9F%9A%A7-Labels?node-id=0%3A1&viewport=-26%2C1115%2C1)

**Comment by meredithxie1995** [comment]
![Screen Shot 2021-08-10 at 12 20 39 PM](https://user-images.githubusercontent.com/84340484/128921924-982abbb6-4655-4777-aa7a-011aa99ab021.png)

Can we add this label for "expired" certification too? For context, we have "active" "expired" and "legacy" types.
